package.json

```diff
@@ -111,5 +111,8 @@
         "regenerator": true
       }]
     ]
+  },
+  "dependencies": {
+    "common-tags": "^1.4.0"
```

Comment:
Actually that was a question for you :) 

I think it would be nice to separate what's needed for development from what's needed for the app, which is the initial goal of these 2 properties. I believe that we can technically put everything like we want thanks to webpack and its tree-shaking algorithm but I don't really like it. (maybe that's just my habit though)

As a side note, debugger.html put everything in `dependencies` :) which I don't like either :p